source/core/Math.cpp

```diff
@@ -899,7 +899,7 @@ DECLARE_VIREO_PRIMITIVE3( Scale2XComplexSingleInt8, ComplexSingle, Int8, Complex
     _Param(2).real(Scale2X_SingleInt32(_Param(0).real(), Int32(_Param(1))));
     _Param(2).imag(Scale2X_SingleInt32(_Param(0).imag(), Int32(_Param(1))));
 } )
-DECLARE_VIREO_PRIMITIVE3( PolarComplexSingle, Single, Single, ComplexSingle, (_Param(2) = _Param(1) == 0.0 ? ComplexSingle(_Param(0),0.0) : _Param(0) >= 0.0 ? polar(_Param(0), _Param(1)) : polar(-_Param(0), _Param(1)+Single(M_PI))) )
+DECLARE_VIREO_PRIMITIVE3( PolarComplexSingle, Single, Single, ComplexSingle, (_Param(2) = _Param(1) == 0.0f ? ComplexSingle(_Param(0),0.0) : _Param(0) >= 0.0f ? polar(_Param(0), _Param(1)) : polar(-_Param(0), _Param(1)+Single(M_PI))) )
```

Comment:
I did this just because sanmut requested it, but I did some testing and:
There is no difference at all in the code generated by emscripten based on whether you use an 'f' suffix for float literals.
On native, the compiler is smart enough to not emit  double if the constant is exactly representable as float, and automatically keeps it float.  So we never need to say 0.0f or 1.0f.